docs/3.0.x/install-rn.mdx

```diff
@@ -5,46 +5,84 @@ title: Install in React Native project
 
 import { TileLink } from "../../src/components";
 
-### Create a new project
+#### Create a new project
+
 
 ```bash
-npx react-native init AwesomeNativeBase
+  npx react-native init AwesomeNativeBase
 cd AwesomeNativeBase
 ```
 
-### Install dependencies
 
-<DocTabs
-defaultValue='yarn'
-values={[
-{ label: 'yarn', value: 'yarn' },
-{ label: 'npm', value: 'npm' }
-]}>
 
-<DocTabItem value="yarn">
+#### Install dependencies
+
+<Box mt='3' />
+<Box 
+ _dark={{ bg: "#171E2E" }}
+_light={{ bg: "coolGray.50" }}
+rounded='lg' 
+py="3"
+px="4"
+pb='4'
+rounded="md"
+mb='8'
+>
+<Text bold mb="2" fontSize="md" >
+  Using yarn
+</Text>
+<Text
+  _dark={{ color: "coolGray.300" }}
+  _light={{ color: "coolGray.800" }}
+>
+  Lorem ipsum, dolor sit amet consectetur adipisicing elit. Facere dolore rem iure corrupti vel ab, repellendus temporibus cumque similique ducimus eaque dolor, sequi itaque vitae sed aliquam vero mollitia optio!
```

Comment:
Can you remove these 🙏

Reply:
Hey @flexbox, Thanks for the review. We will be adding content before release.